retrofit/src/main/kotlin/com/smartnsoft/ws/retrofit/CacheException.kt

```diff
@@ -0,0 +1,10 @@
+package com.smartnsoft.ws.retrofit
+
+import java.lang.RuntimeException
+
+/**
+ * 
+ * @author David Fournier
+ * @since 2019.06.19
+ */
+class CacheException(message: String?) : RuntimeException(message)
```

Comment:
Why is this inheriting from RuntimeException ? 🤔

Reply:
it's ok